downstream/modules/eda/proc-eda-set-up-rhaap-credential.adoc

```diff
@@ -0,0 +1,45 @@
+[id="eda-set-up-rhaap-credential"]
+
+= Setting up a {PlatformName} credential
+
+You can create a {PlatformName} credential type to run your rulebook activations.  
+
+.Prerequisites
+
+* You have created a user.
+* You have obtained the URL and the credentials to access {ControllerName}.
+
+
+.Procedure
+
+. Log in to the {PlatformName} Dashboard.
+. From the navigation panel, select {MenuADCredentials}.
+. Go to *Create Credentials*.
+. Insert the following:
++
+Name:: Insert the name.
+Description:: This field is optional.
+Organization:: Click the list to select an organization or select *Default*.
+Credential type:: Click the list and select *{PlatformName}*. 
++
+[NOTE]
+====
+When you select the credential type, the *Type Details* section is displayed with fields that are applicable for the credential type you chose.   
+====
+. In the required {PlatformName} field, enter your automation controller URL. 
++
+[NOTE]
+====
+For {EDAcontroller} 2.5 with {ControllerName} 2.4, use the following example: https://<your_controller_host>
+
+For {PlatformNameShort} {PlatformVers}, use the following example: https://<your_gateway_host>/api/controller/
+====
+. Enter a valid username and password or an Oauth Token. 
```

Comment:
**T**oken - > **t**oken  (should be in small case, if it isn't a proper noun).

Reply:
This is how it's displayed in the UI. I should probably capitalize all of these fields since that's how they display. 
```suggestion
. Enter a valid Username and Password or Oauth Token. 
```